src/Microsoft.Kusto.ServiceLayer/QueryExecution/Contracts/ResultSetSummary.cs

```diff
@@ -40,7 +43,38 @@ public class ResultSetSummary
         /// </summary>
         public SpecialAction SpecialAction { get; set; }
 
-        public override string ToString() => $"Result Summary Id:{Id}, Batch Id:'{BatchId}', RowCount:'{RowCount}', Complete:'{Complete}', SpecialAction:'{SpecialAction}'";
+        public VisualizationOptions Visualization { get; set; }
```

Comment:
I think we've typically been trying to consistently use the "// <summary>" doc comments for the Contract definitions.